fiftyone/core/view.py

```
@@ -447,6 +450,70 @@ def view(self):
+            if aggregate_func is not None:
+                if callable(aggregate_func):
+                    result = aggregate_func(
+                        [future.result() for future, _ in futures]
+                    )
+                else:
+                    raise ValueError("aggregate_func must be callable")
+            else:
+                result = []
+
+                # Collect results
+                for future, sample in futures:
```

Ruff (0.8.2)

Loop control variable `sample` not used within loop body

(B007)